base_comment_template/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 NextERP Romania SRL
```

Comment:
Remove this, but add the corresponding script for this version, taking into account that you can come from 13.0.2.0.0.